File: fix_join.js
const fs = require('fs');
let code = fs.readFileSync('src/compiler/core/promptAssembler.ts', 'utf8');
code = code.replace(/\.join\("[\s\S]*?"\)/, '.join("\\n")');
fs.writeFileSync('src/compiler/core/promptAssembler.ts', code);

File: fix_join.py
with open('src/compiler/core/promptAssembler.ts', 'r') as f:
    code = f.read()

import re
code = re.sub(r'\)\`\)\.join\("\n"\)', ')`).join("\\n")', code)
with open('src/compiler/core/promptAssembler.ts', 'w') as f:
    f.write(code)
